### PR TITLE
created models and repos

### PR DESCRIPTION
**Pull request by pdPinho**
Added basic models and repos

### CODE REVIEW

File: src/main/java/tqs/project/api/Models/Bebida.java
Comment (WildBunnie) [inline src/main/java/tqs/project/api/Models/Bebida.java:1]: I don't think the macros need to not be nullable

File: src/main/java/tqs/project/api/Models/Prato.java
Comment (WildBunnie) [inline src/main/java/tqs/project/api/Models/Prato.java:1]: same thing with the macros not needing to be not nullable

File: src/main/java/tqs/project/api/Models/Reserva.java
Comment (WildBunnie) [inline src/main/java/tqs/project/api/Models/Reserva.java:1]: at least the date should be nullable i think

File: src/main/java/tqs/project/api/Models/Bebida.java
Comment (pdPinho) [reply src/main/java/tqs/project/api/Models/Bebida.java:1]: We need to keep that in mind during frontend's UI creation then

File: src/main/java/tqs/project/api/Models/Reserva.java
Comment (pdPinho) [reply src/main/java/tqs/project/api/Models/Reserva.java:1]: should be or shouldn't?

Comment (WildBunnie) [reply src/main/java/tqs/project/api/Models/Reserva.java:1]: not nullable*

**Review by WildBunnie — APPROVED** [review]
LGTM 👍

**Pull request merged**